app/api/search/artist/route.ts

```diff
@@ -0,0 +1,26 @@
+import { NextRequest } from "next/server";
+import { getArtists } from "@/lib/supabase/in_process_artists/getArtists";
+
+export async function GET(req: NextRequest) {
+  const query = req.nextUrl.searchParams.get("query") || "";
+  if (!query || query.trim().length === 0) {
+    return Response.json({ artist: null });
```

Comment:
Please make this response more helpful to let the sender know they are missing a valid `query` param
```suggestion
    return Response.json({ error: "Missing param: query" }, { status: 400 });
```